proj/src/main/webapp/README.md

```diff
@@ -0,0 +1,12 @@
+$ npm install browserify -g
+$ cd step14-2020/proj/src/main/webapp
+$ npm install
+$ cd ../../..
```

Comment:
Maybe mention that the npm install steps are a one time thing while the browserify and mvn commands below are supposed. to be called each time you want to run your local code.